src/test/system_tests/ceph_s3_tests/test_ceph_s3.js

```diff
@@ -100,29 +99,35 @@ async function test_worker() {
     }
 }
 
-async function run_single_test(test) {
-    let ceph_args = `S3TEST_CONF=${CEPH_TEST.test_dir}${CEPH_TEST.ceph_config}`;
-    if (S3_CEPH_TEST_SIGV4.includes(test)) {
+async function run_single_test(test_name) {
+    let ceph_args = `S3TEST_CONF=${process.cwd()}/${CEPH_TEST.test_dir}${CEPH_TEST.ceph_config}`;
+    if (S3_CEPH_TEST_SIGV4.includes(test_name)) {
         ceph_args += ` S3_USE_SIGV4=true`;
     }
-    let base_cmd = `${ceph_args} ./${CEPH_TEST.test_dir}${CEPH_TEST.s3_test_dir}virtualenv/bin/nosetests`;
-    if (!S3_CEPH_TEST_OUT_OF_SCOPE_REGEXP.test(test)) {
+    let base_cmd = `${ceph_args} tox ${TOX_ARGS}`;
+    if (!S3_CEPH_TEST_OUT_OF_SCOPE_REGEXP.test(test_name)) {
         try {
-            const test_name = test.replace(S3_CEPH_TEST_STEMS_REGEXP, pref => `${pref.slice(0, -1)}:`); //Match against the common test path
             if (test_name.includes('boto')) {
-                base_cmd = `${ceph_args} ./${CEPH_TEST.test_dir}${CEPH_TEST.s3_test_dir}virtualenv/bin/nosetests -v -s -A 'not fails_on_rgw'`;
+                base_cmd = `${ceph_args} tox ${TOX_ARGS} -- -m 'not fails_on_aws'`;
```

Comment:
Why did you change it from 'not fails_on_rgw' to 'not fails_on_aws'?
Please add it to the changes in the description.

Reply:
it was changed in the s3-tests:
https://github.com/ceph/s3-tests/pull/482/files#diff-7b3ed02bc73dc06b7db906cf97aa91dec2b2eb21f2d92bc5caa761df5bbc168fR44-R48

I'll add that in the description

Reply:
Thank you for adding the link.
I still don't understand this change, I send this question ([here](https://github.com/ceph/s3-tests/pull/482/files#r1226106436)).